content/docs/user-guide/assets/pipeline/asset-builders.md

```diff
@@ -38,3 +38,58 @@ Process Job generates the product asset and product dependencies. The Asset Buil
 `ProcessJobResponse` places the product assets in a temp directory. Asset Processor moves the product assets to the **Asset Cache** and populates the **Asset Database** and **Asset Catalog** with information it uses to track the product assets, product dependencies, and the jobs that produced them.
 
 Process Job is multithreaded. Several Asset Builders can run multiple process jobs simultaneously.
+
+## Asset Builder Tips and Tricks
+
+### Loading Other Files
```

Comment:
Having these two headings back-to-back looks a bit odd on page. Suggest removing the Tips and Tricks H2 and moving all subsequent headings up a level. Also suggesting sentence casing for headings.

```suggestion
## Loading other files
```